COLOR.js

```diff
@@ -1,35 +1,51 @@
-
 class Color {
-    constructor(colorCode) {
-      this.colorCode = colorCode;
-    }
-    static randomColor() {
-      return new Color(random_hex())
-    }
+  constructor(colorCode) {
+    this.colorCode = colorCode;
+  }
+  static randomColor() {
+    return new Color(random_hex());
+  }
 
-    //add compliment method
-    getCompliment(){
-      let hex =this.colorCode;
-     let compliment_hex= hex.slice(2) + hex.slice(0,2);
-      console.log(compliment_hex)
-      return compliment_hex;
-      //onsubmit create a new color box with complimentary color
-    }
+  isDark() {
+    //https://stackoverflow.com/questions/12043187/how-to-check-if-hex-color-is-too-black
+    //var c = c.substring(1); // strip #
+    var c = this.colorCode;
```

Comment:
So `var` is no longer recommended and you don't see it really in ES6 code – you should use `let` or `const` to signal whether or not you intend to reassign that variable later.